apps/issuer/api.py

```diff
@@ -1035,7 +1039,60 @@ def post(self, request, **kwargs):
                 {"error": "An unexpected error occurred"},
                 status=HTTP_400_BAD_REQUEST
             )
-        
+
+class StaffRequestList(BaseEntityListView):
+    model = IssuerStaffRequest    
+    v1_serializer_class = IssuerStaffRequestSerializer
+    permission_classes = [
+        IsServerAdmin
+        | (AuthenticatedWithVerifiedIdentifier & BadgrOAuthTokenHasScope & ApprovedIssuersOnly)
+    ]
+    valid_scopes = ["rw:issuer"]
+
+    @apispec_delete_operation('IssuerStaffRequest',
+        summary="Delete multiple issuer staff requests",
+        tags=["Issuer staff requests"],
+    )
+
+    def post(self, request, **kwargs):
+        try: 
+            ids = request.data.get('ids', [])  
+
+            with transaction.atomic():
+                    deletion_queryset = IssuerStaffRequest.objects.filter(
```

Comment:
this should also check if the user is part of the institutions that the requests are deleted from, or is that handled elsewhere?


this also applies to some of the requests below, I think.